[CIT-522] update the mqtt port document (#398)
thin-edge/thin-edge.io: docs/src/howto-guides/008_config_local_mqtt_port.md

Using `tedge` command connect to the desired cloud as below.

> This will force all the services (mosquitto, tedge-mapper-c8y.service,tedge-mapper-az.service sm_agent,

Reviewer: Just very minor grammar.

```suggestion
This will force all the services (mosquitto, tedge-mapper-c8y.service, tedge-mapper-az.service, sm_agent,
```

Author: fixed

> This will force all the services (mosquitto, tedge-mapper-c8y.service, tedge-mapper-az.service, sm_agent,

Reviewer 2: ```suggestion
This will configure all the services (mosquitto, tedge-mapper-c8y.service, tedge-mapper-az.service, sm_agent,
```

> tedge-agent.service, tedge-mapper-sm-c8y.service) to use newely set port.

Reviewer 2: ```suggestion
tedge-agent.service, tedge-mapper-sm-c8y.service) to use the newly set port.
```
